src/components/select-field/BaseSelectField.js

```diff
@@ -7,8 +7,10 @@ import { scrollIntoView } from '../../utils/dom';
 import IconCheck from '../../icons/general/IconCheck';
 import SelectButton from '../select-button';
 import DatalistItem from '../datalist-item';
+import PopperComponent from '../popper';
+import SelectFieldDropdown from './SelectFieldDropdown';
 import type { SelectOptionValueProp, SelectOptionProp } from './props';
-import { OVERLAY_WRAPPER_CLASS } from '../../constants';
+import { PLACEMENT_BOTTOM_END, PLACEMENT_BOTTOM_START } from '../popper/constants';
```

Comment:
Can we hide these details inside the PopperComponent?

Reply:
I think the PopperComponent should be pretty general use, exposing something like a  `isRightAligned` prop to PopperComponent feels a bit too dropdown-y